.pre-commit-config.yaml

```diff
@@ -1,13 +1,28 @@
 repos:
-  - repo: https://github.com/psf/black
-    rev: 21.7b0
-    hooks:
-      - id: black
-  - repo: https://github.com/pycqa/flake8
-    rev: "3.9.2"
-    hooks:
-      - id: flake8
-  - repo: https://github.com/pycqa/isort
-    rev: "5.8.0"
-    hooks:
-      - id: isort
+- repo: https://github.com/terrencepreilly/darglint
+  rev: v1.5.8
+  hooks:
+  - id: darglint
+    files: jina/
+    exclude: ^(jina/helloworld/|jina/proto/jina_pb2.py|docs/|jina/resources/)
+    args:
+    - --message-template={path}:{line} {msg_id} {msg}
+    - -s=sphinx
+    - -z=full
+    - -v=2
+- repo: https://github.com/pycqa/pydocstyle
```

Comment:
Please revert the changes to this file

Reply:
I reverted the changes. I think this file was not there when I forked the repository and I was using another pre-commit config file. I guess it got fetched in this merge and got overwritten. Sorry for that, I should've checked before committing.